src/transformers/modeling_fsmt.py

```diff
@@ -0,0 +1,1210 @@
+# coding=utf-8
+# Copyright 2020 The Facebook AI Research Team Authors and The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Original implementation: https://github.com/pytorch/fairseq/tree/master/examples/wmt19
+# Authors:
+# - @alexeib Alexei Baevski
+# - @edunov Sergey Edunov
+# - @michaelauli Michael Auli
+# - @myleott Myle Ott
+# - @nng555 Nathan Ng
+# - David Grangier
+# - Kyra Yee
+#
+# Paper: Facebook FAIR's WMT19 News Translation Task Submission https://arxiv.org/abs/1907.06616
+#
+"""PyTorch Fairseq model, ported from https://github.com/pytorch/fairseq/tree/master/examples/wmt19"""
+
+import logging
+import math
+import random
+import warnings
+from typing import Any, Dict, List, Optional, Tuple
+
+import torch
+import torch.nn.functional as F
+from torch import Tensor, nn
+from torch.nn import CrossEntropyLoss
+
+from .activations import ACT2FN
+from .configuration_fsmt import FSMTConfig
+from .file_utils import (
+    add_code_sample_docstrings,
+    add_end_docstrings,
+    add_start_docstrings,
+    add_start_docstrings_to_callable,
+    replace_return_docstrings,
+)
+from .modeling_outputs import BaseModelOutput, BaseModelOutputWithPast, Seq2SeqLMOutput, Seq2SeqModelOutput
+from .modeling_utils import PreTrainedModel
+
+
+logger = logging.getLogger(__name__)
```

Comment:
```suggestion
logger = logging.get_logger(__name__)
```

I realize now that just changing the case is a bit annoying. Maybe we could add an alias in the logging code so that `getLogger` also works. What do you think?

I believe I intentionally decided to change it to snake case for 1) coherence and 2) to make sure we were using the right logging tool, but now I'm starting to have second thoughts.

Reply:
I think you intuitively chose this approach, knowing that a lot of users already have `import logging` and if the API is the same they will be bypassing the project-wide logger setup.

It also allows for a quick `grep getLogger` to see if you are using the wrong logger.

IMHO, your choice is wise.

besides snake case in function names is weird. again, IMHO.

Reply:
Great, thanks for your input. Let's leave it like it is then :).